src/main/resources/styles/app.css

```
@@ -163,102 +330,121 @@
+    -fx-text-fill: white;
+    -fx-font-size: 14px;
+    -fx-font-weight: bold;
 }
+/* =========================
+   LOADING OVERLAY TEXT
+   ========================= */
 
-.external-link-button:hover {
-    -fx-background-color: rgba(255, 255, 255, 0.12);
+.loading-text {
     -fx-text-fill: white;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)